giga_connectome/utils.py

```diff
@@ -26,7 +26,7 @@ def get_bids_images(
         root=bids_dir,
         database_path=bids_dir,
         validate=False,
-        derivatives=True,
+        derivatives=False,
```

Comment:
> test_get_bids_images: this is from a hack to force pybids to index fmriprep outputs, you are a better person to judge if i am doing the right thing here

This should take care of this warning.

This is a flag to index things in the derivative and not to say that the dataset being index is a derivative.